tools/openshift-ci/verification_tests_ci_entrypoint.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/sh
+
+export BUSHSLICER_DEFAULT_ENVIRONMENT="${BUSHSLICER_TEST_ENVIRONMENT}"
+export OPENSHIFT_ENV_$(echo "${BUSHSLICER_TEST_ENVIRONMENT}" | awk '{print toupper($0)}')_HOSTS="${BUSHSLICER_TEST_CLUSTER}:etcd:master:node"
+export OPENSHIFT_ENV_$(echo "${BUSHSLICER_TEST_ENVIRONMENT}" | awk '{print toupper($0)}')_USER_MANAGER_USERS="${BUSHSLICER_TEST_TOKEN}"
```

Comment:
Should this be `":${BUSHSLICER_TEST_TOKEN}"`

Reply:
good catch, thank you!